mashups/smart-home/dist/things/presence-sensor.js

```diff
@@ -0,0 +1,66 @@
+"use strict";
+/********************************************************************************
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
```

Comment:
```suggestion
 * Copyright (c) 2024 Contributors to the Eclipse Foundation
```

Since it is new?

Reply:
Fixed [here](https://github.com/eclipse-thingweb/test-things/pull/39/commits/28d3fca73fba88444fdfd95ec9cb9aa3a08ac693).